pages/dkp/kaptain/1.3.0/install/konvoy-dkp/index.md

```diff
@@ -181,5 +184,7 @@ Once all components have been deployed, you can log in to Kaptain:
 [install-spark-dkp2]: /dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/spark-operator/
 [install-spark-konvoy1]: /dkp/kommander/1.4/projects/platform-services/platform-services-catalog/kudo-spark/
 [kommander-install]: /dkp/kommander/latest/install/
+[kommander-gpu]: /dkp/kommander/latest/gpu/
```

Comment:
We'll need to change this - `/latest/gpu` isn't valid ... Let me get a good link here for you.

Reply:
WAIT - nm - the redirect locally is broken, but it works in real life.